src/perform_step/bdf_perform_step.jl

```diff
@@ -653,8 +653,6 @@ function perform_step!(integrator,cache::QNDFConstantCache,repeat_step=false)
       end
       backward_diff!(cache,D,D2,k)
     end
-  else
```

Comment:
what happens during the else now?

Reply:
The gamma value should not be updated before `cnt>k`, the code was doing that previously. You can see the same results in QNDF1. (Current Implementation gives 1st order estimate only which needs to be checked)